style.css

```diff
@@ -71,5 +76,239 @@ h1 {
   background-image: url(image/image_geometry_2.jpg);
   background-repeat: no-repeat;
   background-size: 100%;
-  height: 610px;
+  height: 260px;
+}
+
+h2 {
+  font-family: 'Playfair Display', sans-serif;
+  margin-left: 5%;
+  font-size: 50px;
+}
+
+.Works-section {
+  display: grid;
+}
+
+.worksection1 {
+  background-image: url(image/Rectangle21.jpg);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.worksection2 {
+  background-image: url(image/Rectangle22.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.worksection3 {
+  background-image: url(image/Rectangle23.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.worksection4 {
+  background-image: url(image/Rectangle24.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.worksection5 {
+  background-image: url(image/Rectangle25.png);
+  background-repeat: no-repeat;
+  background-size: 100%;
+  height: 260px;
+  display: grid;
+}
+
+.rectangle {
+  background-color: #1a2236;
+  height: 156px;
+}
```

Comment:
- [ ] As the previous reviewer mentioned, please make sure your design matches the figma design. kindly fix the spacing, margin, and padding.
# your design 👇
![finish](https://user-images.githubusercontent.com/66992901/191821020-f4984bea-8741-4145-a021-93819d1bd7f0.PNG)
# Figma design 👇
![finishfigma](https://user-images.githubusercontent.com/66992901/191821071-eda0ecbb-dd67-4575-a0a9-c96693b44abd.PNG)